bin/__tests__/eslint-focus-test.mjs

```diff
@@ -146,13 +147,11 @@ test("--fix --fix-type add-disable-directive", async () => {
 	).resolves.toMatchInlineSnapshot(`
 		"// eslint-disable-next-line eqeqeq
 		if (Math.random() == 1) {
-		// eslint-disable-next-line capitalized-comments
-			// one branch
```

Comment:
This seems wrong, no?

Reply:
It changed by itself while updating snapshots. Should I change it?

Reply:
addressed it! it took me 2 years to solve the issue :d